LDAP/disable_deactivated_accounts.py

```diff
@@ -64,6 +64,15 @@ def list_active_users(users):
             ret_users.append(user)
     return ret_users
 
+# also need a list of blocked users to see if they should be unblocked
+def list_blocked_users(users):
+    ret_users = []
+    for user in users:
+    	if u'blocked' in user:
+            if user[u'blocked'] is True:
```

Comment:
recommend using `user.get(u'blocked')` for reliability (will work even if the field is missing)